docs/tutorials/articles/colab_with_ngrok/index.md

```diff
@@ -115,14 +115,21 @@ To address this issue, you can modify the *change_delay* parameter in one of the
 
 - **Locally**: You can make changes to this parameter locally, directly within the Taipy visual
   element.
-  ```python
-  <|{text}|input|change_delay=800|>
-  ```
+
+=== "Markdown" 
+   ```python
+   <|{text}|input|change_delay=800|>
+   ```
+=== "Python"
+   ```python
+   tgb.input("{text}", change_delay=800)
+   ```
```

Comment:
Must be indented if it is part of the first bullet.

```suggestion
    === "Markdown" 
       ```python
       <|{text}|input|change_delay=800|>
       ```
    === "Python"
       ```python
       tgb.input("{text}", change_delay=800)
       ```
```

Reply:
Will this work? I don(t know what is the difference

Reply:
It should